omod/src/main/resources/webModuleApplicationContext.xml

```diff
@@ -25,4 +25,23 @@
 	<bean class="org.openmrs.ui.framework.StandardModuleUiConfiguration">
 		<property name="moduleId" value="ugandaemrpoc"/>
 	</bean>
+	<bean class="org.openmrs.ui.framework.page.GlobalResourceIncluder">
+		<property name="resources">
+			<list>
+				<bean class="org.openmrs.ui.framework.resource.Resource">
+					<property name="category" value="css" />
+					<property name="providerName" value="${project.parent.artifactId}" />
+					<property name="resourcePath" value="css/ugandaemr-poc-styles.css"/>
+					<property name="priority" value="-500" />
+				</bean>
+
+				<bean class="org.openmrs.ui.framework.resource.Resource">
```

Comment:
@dbaluku Rename this to ugandaemr-poc.js as it should not override aijar.js

Reply:
Done